docs/devel/RELEASE.md

```diff
@@ -118,3 +118,57 @@ CHANGELOG.md snippet and uploading the binaries from the build phase.
 		```bash
 		curl -X POST -d {} https://api.netlify.com/build_hooks/5cc3aa86495f22c7a368f1d2
 		```
+
+# Bugfix Release Process
+
+If this is the first bugfix for the release, create the release branch from the
+release tag:
+
+```bash
+git checkout -b release-0.14 v0.14.0
+```
+
+Otherwise, checkout the release branch and rebase on upstream:
+
+```bash
+git fetch upstream
+git checkout release-0.14
+git rebase upstream/release-0.14
+```
+
+Cherry pick the changes from master or other branches onto the bugfix branch:
+
+```bash
+git cherry-pick <commit-id>
```

Comment:
nit: We might want to use `-x` to make it a little easier to keep track of them

Reply:
Fixed